python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -399,8 +399,8 @@ public void visitCompoundAssignment(CompoundAssignmentStatement pyCompoundAssign
     public void visitGlobalStatement(GlobalStatement pyGlobalStatementTree) {
       pyGlobalStatementTree.variables()
         .forEach(name -> {
-          currentScope().addGlobalName(name.name());
           moduleScope.addBindingUsage(name, Usage.Kind.GLOBAL_DECLARATION, null);
```

Comment:
I know it's not really related to this PR, however I think we should update as well `Usage#isBindingUsage()` to avoid considering `Usage.Kind.GLOBAL_DECLARATION` as a binding usage. wdyt?